### ISSUE TITLE
Markdown editor and preview scroll jumping when editing with images

### ISSUE THREAD

**Issue by adevade** [Microsoft/vscode#49150]
- VSCode Version: 1.22.2 && 1.23.0
- OS Version: Windows 10 with the latest updates

Steps to Reproduce:

1. Create a new markdown document and fill it with content. Enough to scroll a bit.
2. Enable live preview split.
3. Add an image near the top (I've only tested with those "build badges")
4. Scroll down and start writing again. Saving also makes the content jump.
5. Both the editor and the preview start to jump around.

Does this issue occur when all extensions are disabled?: Yes

### Video of the bug
![2018-05-04_00-09-37](https://user-images.githubusercontent.com/1066486/39605205-ae913626-4f2f-11e8-8a2f-fcf8dc257128.gif)

This is the image I added in the demo, but it did it with every other "badge" I tested as well.
```
[![npm](https://secure.travis-ci.org/emberjs/ember.js.svg?branch=master)](http://travis-ci.org/emberjs/ember.js)
```

**Comment by StephenWeatherford** [comment]
This is incredibly annoying in 1.23.0.k However, I can't seem to repro it in 1.23.0-insider.

**Comment by ipatalas** [comment]
Experiencing exactly the same on:
```
Version 1.23.1
Commit d0182c3417d225529c6d5ad24b7572815d0de9ac
Date 2018-05-10T17:11:38.209Z
Shell 1.7.12
Renderer 58.0.3029.110
Node 7.9.0
Architecture ia32
```

This is so annoying I have to turn off preview when writing because otherwise active line quickly goes out of screen.

**Comment by StephenWeatherford** [comment]
Repros for me in certain conditions (insiders and current).  E.g.:

1) Download the following file and open it in vscode: https://github.com/Microsoft/vscode-cosmosdb/blob/master/README.md
2) Move cursor to middle of file e.g. right after this line:

  ![Mongo Scrapbook](resources/Scrapbook.gif)

3) Scroll the editor so that this line is about in the middle of the screen (otherwise doesn't repro)
4) Open preview (or earlier)
5) Start typing
!) Both windows scroll

![preview-jumping](https://user-images.githubusercontent.com/6913354/40022251-3c57daba-577c-11e8-90a4-af4f2d47de6c.gif)

**Comment by ThadHouse** [comment]
I was able to reproduce this with AppVeyor build status images. It was really annoying, as my editor was just scrolling off the screen.

**Comment by jumpkick** [comment]
Still happening for me with:
Version 1.24.1
Commit 24f62626b222e9a8313213fb64b10d741a326288
Date 2018-06-13T17:51:32.889Z
Shell 1.7.12
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64

I wouldn't mind if the only preview windows was jumping around, but using the editor pane at the end of a document with the preview pane open jumps the editors line up and down on and off the screen.

Workaround is to close preview while editing.

**Comment by mrchief** [comment]
I was going to report this but found this issue. This issue makes markdown editing unusable. In the example gif, I'm just inserting a space and deleting it. With every keystroke, the editor scrolls and just in 4 strokes, it scrolls out of sight.

![vscode jumpy markdown](https://user-images.githubusercontent.com/781818/41753223-0692e42e-759a-11e8-9169-3be62508607b.gif)


Version 1.24.1
Commit 24f62626b222e9a8313213fb64b10d741a326288
Date 2018-06-13T17:51:32.889Z
Shell 1.7.12
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64

**Comment by rmariuzzo** [comment]
I'm facing this issue as well. I have just learnt that images causes this as explained here.

**Comment by detj** [comment]
Still happening on `v1.24.1` stable. Pretty annoying indeed. Is there any setting to turn off the automatic scrolling completely ?

**Comment by slikts** [comment]
The markdown preview is very frustrating to use due to this issue; I've been forced to comment out images while editing.

**Comment by mrchief** [comment]
@slikts You can close the default preview pane and use an extension instead. Less editing pain that way.

**Comment by copleykj** [comment]
This seems to have resolved for me with the 1.28.0 update

**Comment by mjbvz** [comment]
Duplicate of #56674

If you still see this, please open a new issue with some example markdown and I'll take a look
